omnibus/config/software/datadog-agent.rb

```diff
@@ -43,24 +42,20 @@
     env = {
         'GOPATH' => gopath.to_path,
         'PATH' => "#{gopath.to_path}/bin:#{ENV['PATH']}",
-        "Python2_ROOT_DIR" => "#{windows_safe_path(python_2_embedded)}",
         "Python3_ROOT_DIR" => "#{windows_safe_path(python_3_embedded)}",
-        "CMAKE_INSTALL_PREFIX" => "#{windows_safe_path(python_2_embedded)}",
+        "CMAKE_INSTALL_PREFIX" => "#{windows_safe_path(python_3_embedded)}",
```

Comment:
was `CMAKE_INSTALL_PREFIX` always set using python 2 embedded path 😮 ?

Reply:
It seems that way, I'm unsure of the consequences from this typo TBH